README.md

```diff
@@ -66,20 +66,34 @@ You can test the pipeline with a small example dataset using Hydra configuration
 
 ### Example: Using Custom Data Paths
 
+We demonstrate how to extract embeddings using the Abdomen CT model using a public example from the Merlin dataset:
 ```bash
 uv run rate-extract \
     --model pillar0 \
     --dataset abd_ct_merlin \
     --split train \
     --batch-size 4 \
-    --output-dir cache/pillar0_abd_ct_merlin \
     --model-repo-id YalaLab/Pillar0-AbdomenCT \
     --ct-window-type all \
-    --modality abdomen_ct \
+    --output-dir cache/pillar0_abd_ct_merlin \
     data.train_json=data/merlin_example/train.json \
     data.cache_manifest=data/merlin_example/manifest.csv
 ```
 
+To extract vision embeddings using the CT models, please refer to the example metadata in [data/rve_example](data/rve_example/). 
+```bash
+uv run rate-extract \
+    --model pillar0 \
+    --dataset rve_chest_ct # rve_abd_ct, rve_brain_ct, rve_chest_ct \
```

Comment:
slash is before the comment

--dataset rve_chest_ct \ # rve_abd_ct, rve_brain_ct, rve_chest_ct